setup/security/authorization/index.md

```diff
@@ -68,17 +68,41 @@ accounts.
 ### Accounts
 In the dark ages (before Fiat), only accounts could be restricted. Because
 Clouddriver is the source of truth for accounts, Fiat reaches out to Clouddriver
-to gather the list of available accounts. To add access restrictions to an
-account, update your `halconfig` to include the `requiredGroupMembership` field.
+to gather the list of available accounts. 
+
+There are two types of access restrictions to an account, `READ` and `WRITE`. Users must have
+matching roles to all the `READ` permissions of an account to view the account's cloud resources.
+They must have matching roles to all the `WRITE` permissions to make changes to the resources.
+
+These halyard commands manage the `READ` and `WRITE` permissions.
 
 ```bash
 PROVIDER= # Your cloud provider
-GROUP=    # The new group membership
 
-hal config provider $PROVIDER account edit $ACCOUNT --add-required-group-membership $GROUP
+hal config provider $PROVIDER account edit $ACCOUNT \
+  --add-read-permission role1 \ # Adds a READ permission
+  --add-write-permission role2 \ # Adds a WRITE permission
+  --remove-read-permission role3 \ # Removes a READ permission
+  --remove-write-permission role4 # Removes a WRITE permission
+
+# Alternatively, you can overwrite the whole read or write list, comma delimited.
+hal config provider $PROVIDER account edit $ACCOUNT \
+  --read-permissions role1,role2,role3 \
+  --write-permissions role1,role2,role3
 ```
 
-Alternatively, you can overwrite the whole list using the `--required-group-membership` flag.
+(Deprecated) `requiredGroupMembership` is the old way to add access restrictions to an account. 
+This method does not distinguish between `READ` and `WRITE`.
```

Comment:
@stewchen 
Just out of curiosity, what was the effect when we didn't distinugish between read and write? User would have both?

Reply:
Yup

Reply:
Do we know at which version of spinnaker this stopped being accepted? Maybe we can call that out here

Reply:
Actually, I misspoke - they are still functional - but I guess people are unhappy with the deprecation messages introduced in https://github.com/spinnaker/clouddriver/pull/1642. How can I find the clouddriver version and/or spinnaker release that introduced that commit?

Reply:
In the commit, it lists what tags this change was included with:

![yfftd8sq4q3](https://user-images.githubusercontent.com/13141550/40795386-ed6a9c88-64cf-11e8-94a1-78ca8cc51ce8.png)

It appears this code has been included since the 1.0.0 release. Hooray!